tests/src/Traits/ClientTestTrait.php

```diff
@@ -34,13 +34,26 @@ protected function getTestingClient(array $configuration = [], array $responseQu
         $handlerStack = HandlerStack::create(new MockHandler($responseQueue));
         $handlerStack->push(Middleware::history($this->clientHistory));
 
+        $defaultConfiguration = [
```

Comment:
After introducing the config trait, it turned out that the client sometimes accepted empty configuration. This is why I moved it here. This way, it is always added to the test client.